docs/en/connector-v2/sink/Redis.md

```diff
@@ -49,13 +52,17 @@ Upstream data is the following:
 | 200  | get success    | true    |
 | 500  | internal error | false   |
 
-If you assign field name to `code` and data_type to `key`, two data will be written to redis:
-1. `200 -> {code: 200, message: true, data: get success}`
-2. `500 -> {code: 500, message: false, data: internal error}`
+You can customize the Redis key using '{' and '}', and the field name in '{}' will be parsed and replaced by the field value in the upstream data. For example, If you assign field name to `{code}` and data_type to `key`, two data will be written to redis:
+1. `200 -> {code: 200, data: get success, success: true}`
+2. `500 -> {code: 500, data: internal error, success: false}`
```

Comment:
So the default behavior had change? Before this PR,
`key = code` will set value as `200 -> {code: 200, data: get success, success: true}`.
But after this change, it would be `code -> {code: 200, data: get success, success: true}`
We need to be backward compatible.

Reply:
we can introduce new key name like `key_field` to avoid this break change happend.

Reply:
Ok, I will do it later.

Reply:
If we introduce new key name, such as keyfield, it will cause users to write two keys, namely key and keyfield, which is very easy to confuse because key is a required field. I think we can introduce a new option to control whether custom keys are supported.

Reply:
we can mark `key` as deprecated.

Reply:
cc @FuYouJ

Reply:
Can we add a configuration option to decide whether the key is a specific constant value or a variable value?
@Hisoka-X @lm-ylj

Reply:
I prefer add new config key. Then we can remove old key in the future rather than maintain two keys.

Reply:
WDYT? @hailin0 @liugddx